### DISCUSSION TITLE
(Pagamentos internacionais - cobrança em moeda estrangeira.) Qual melhor gateway de pagamento?

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1767] [Discussão]
Olá galera, estou participando do desenvolvimento de uma aplicação e pretendemos atingir um publico internacional, e peço dicas, qual melhor gateway para realizar cobranças estrangeiras?  que tenha conversão de moedas e coisas do tipo, e tenha compatibilidade com o brasil.

Obs: Integração com react/ionic

**Comment by giovannism20** [comment]
Bom dia @ViniciusGularte,

Nunca cheguei a trabalhar com cobranças internacionais mas sei que levamos em conta taxas e diversas outras coisas (pois já trabalhei com pagamento no Brasil), um serviço que vejo ser usado com bastante frequência é o [EBANX](https://www.ebanx.com/br/). A própria [PlayStation](https://www.playstation.com/pt-br/), [Spotify](https://www.spotify.com/br/).

**Comment by willycamargo** [comment]
Implementei [stripe](https://stripe.com/) uma vez, uns 3 anos atrás... Foi bem simples mas não suportava boleto na época.

**Comment by lucasctd** [comment]
@willycamargo Demorou muito pra tu conseguir a conta? Eu já solicitei convite duas vezes e até agora nada.
Eu uso o PayPal pra receber pagamentos de fora mas acho a taxa deles muito alta (6,4% + um valor fixo) fora que a taxa de conversão deles é muito baixa tb...
Tentei usar o 2checkout mas eles transferem pra bancos do Brasil em dolar, então vou acabar pagando taxa 2x.
Eu queria uma plataforma que me deixasse enviar invoices para americanos e que fizesse a conversão pra real, só que com uma taxa mais baixa que o PayPal.

**Comment by lucasctd** [comment]
Uma solução que tentei foi receber o dinheiro na minha conta de banco dos EUA, aí eu poderia transferir pra cá usando a transferwise, que tem uma taxa infinitamente mais barata que o PayPal. O problema é que todos os serviços de pagamento que tentei (americanos), solicitavam SSN (Um CPF de lá), que é algo que não tenho xD.

**Comment by lucashfreitas** [comment]
Cara, ja utilizei bastantes soluçōes tanto nacionais e internacionais, e nenhuma delas chegou PERTO do [stripe](https://stripe.com/en-br/pricing). 

A ferramenta è maravilhosa, pratica e foi realmente feita pensando em facilitar a vida dos desenvolvedores. Com o stripe voce consegue cobrar em qualquer currency e ele mesmo faz a conversão pra sua moeda/conta bancaria. 

Venho utilizando largamente e não tive nenhum problema. Na minha opinião pessoal, nunca vi um serviço de pagamento que oferece suporte, aplicação, API/Documentação igual o stripe.

Nunca utilizei ele no Brazil, mas parece que já está disponível por ai. Da uma olhada!!

Qualquer dúvida manda um inbox.

**Comment by ViniciusGularte** [comment]
Pois é ouço/vejo muito bem do **stripe**, até grande parte dos exemplos na web utilizam ele pela facilidade de integração, problema que no Brazil  é só por convite, tinha solicitado ano passado e este ano e nada ainda :disappointed:

**Comment by lucashfreitas** [comment]
Entendi, tem uma opção pra voce requisitar o convite, quem sabe você consegue? :)

Boa sorte

**Comment by lucasctd** [comment]
@ViniciusGularte mesmo caso aqui, também solicitei ano passado e este ano, mas nunca recebi um retorno. Eles recomendam o Stripe Atlas rs, mas são $500 pra abrir uma empresa lá, sem condições.
